L2-Zad.3-Kalkulator/src/test/java/project/CalculatorTest.java

```diff
@@ -0,0 +1,50 @@
+package project;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CalculatorTest {
```

Comment:
super, że trzymasz konwencje given when then oraz te napisane testy sa ok. Nazwy metod mogłby być jeszcze bardziej opisujace co dokładnie testujesz. W testach staramy się też przetestować też warunki brzegowe/corner case. Czyli staramy sie napisac testy wykorzystujac takie dane ktore mogłby "zepsuć" działanie programu np przy metodzie ktora dodaje a przyjmuje jako parametry int, sprawdzmy co sie stanie jesli np jeden parametr bedzei mial wartosc równą maksymalnej dla integera. Przy dzieleniu, spróbujmy podzielić przez zero itd. Czyli podsumpwując myslę, że dla każdej metody z kalkulatora możemy napisac przynajmiej 3 testy. Pierwszy sprawdzający tzw happy path (czyli jak u Ciebie, sprawdzający standardową ścieżkę) oraz dwa pozostałe testy które testują zachowanie metod dla waunków brzegowych

Reply:
Poprawione, ale test na dzielenie przez zero nie działa (o dziwo :) Nie wiem nawet co tam wpisać w expected, "Infinity" się nie da :)

Reply:
a to może w takim razie w metodzie trzeba to jakoś "obsłużyć" ? np sprawdzić czy chcemy dzielić przez zero ? :)